input/includes/value-sets.md

```diff
@@ -0,0 +1,401 @@
+<table border="1" cellspacing="0" cellpadding="0" width="100%">
+<thead>
+  <tr style="background-color: #f2f2f2;">
```

Comment:
Overall doesn't describe which location to get this content. As in, is this AU Base, is the the FHIR terminology, is it NCTS, is it AU Core, is it somewhere else?

The content presented here is a different view of which terminology is used in which profile.

Reply:
Thanks @dt-r. My apologies, I pushed value-sets.md and code-systems.md by mistake. They are not referenced by anything. The content for review is in terminology.md. I will remove these from the PR now.